examples/guestbook/README.md

```diff
@@ -323,70 +372,88 @@ and create the replication controller by running:
 
 ```console
 $ kubectl create -f examples/guestbook/redis-slave-controller.yaml
-replicationcontrollers/redis-slave
+replicationcontroller "redis-slave" created
 
 $ kubectl get rc
-CONTROLLER                             CONTAINER(S)            IMAGE(S)                                 SELECTOR                     REPLICAS
-redis-master                           master                  redis                                    name=redis-master            1
-redis-slave                            slave                   gcr.io/google_samples/gb-redisslave:v1   name=redis-slave             2
+CONTROLLER                  CONTAINER(S)            IMAGE(S)                                 SELECTOR                                      REPLICAS
+redis-master                master                  redis                                    app=redis,role=master,tier=backend            1
+redis-slave                 slave                   gcr.io/google_samples/gb-redisslave:v1   app=redis,role=slave,tier=backend             2
 ```
 
 Once the replication controller is up, you can list the pods in the cluster, to verify that the master and slaves are running.  You should see a list that includes something like the following:
 
 ```console
 $ kubectl get pods
-NAME                                           READY     STATUS    RESTARTS   AGE
+NAME                                READY     STATUS    RESTARTS   AGE
 ...
-redis-master-dz33o                             1/1       Running   0          2h
-redis-slave-35mer                              1/1       Running   0          2h
-redis-slave-iqkhy                              1/1       Running   0          2h
+redis-master-dz33o                  1/1       Running   0          2h
+redis-slave-35mer                   1/1       Running   0          2h
+redis-slave-iqkhy                   1/1       Running   0          2h
 ```
 
 You should see a single redis master pod and two redis slave pods.  As mentioned above, you can get more information about any pod with: `kubectl describe pods/<pod_name>`.
 
-### Step Four: Create the redis slave service
+### Step Five: Set up the guestbook frontend service.
```

Comment:
Again, it would be nice to combine steps 5 and 6, though the discussion about creation of an external load balancer makes that more awkward.